tensorflow/compiler/xla/service/llvm_ir/math_ops.h

```diff
@@ -22,9 +22,25 @@ limitations under the License.
 namespace xla {
 namespace llvm_ir {
 
+enum TanhType {
+  Double = 0,
+  Float = 1,
+};
+
+static constexpr float UPPER_BOUNDER_DOUBLE = 9.0;
```

Comment:
Constant code style is kUpperBoundDouble, probably we could use a more descriptive name? (like, specify tanh).

Does negation deserve it's own constant, or could we negate existing one?